include/swift/Demangling/ManglingUtils.h

```diff
@@ -191,8 +191,8 @@ void mangleIdentifier(Mangler &M, StringRef ident) {
       // of the identifier - that's why we added the dummy-word).
       // The first thing: we add the encoded sub-string length.
       M.Buffer << (Repl.StringPos - Pos);
-      assert(!isDigit(ident[Pos]) &&
-             "first char of sub-string may not be a digit");
+//       assert(!isDigit(ident[Pos]) &&
```

Comment:
Missed this.

Reply:
Thanks. I'm fixing this up along with some other stuff. Not quite done with it yet!